### PR TITLE
planning: open space fixed wrong fallback trajectory

### PR DESCRIPTION
**Pull request by deidaraho**
TODO(Runxin): find a better fallback trajectory.

### CODE REVIEW

File: modules/planning/tasks/deciders/open_space_fallback_decider.cc [hunk 1]
```diff
@@ -33,7 +33,7 @@ OpenSpaceFallbackDecider::OpenSpaceFallbackDecider(const TaskConfig& config)
 
 Status OpenSpaceFallbackDecider::Process(Frame* frame) {
   std::vector<std::vector<common::math::Box2d>> predicted_bounding_rectangles;
-  double collision_distance;
```
Comment (Capri2014) [inline modules/planning/tasks/deciders/open_space_fallback_decider.cc]: Load this one from configs

File: modules/planning/tasks/deciders/open_space_fallback_decider.cc [hunk 2]
```diff
@@ -48,35 +48,25 @@ Status OpenSpaceFallbackDecider::Process(Frame* frame) {
     *(frame_->mutable_open_space_info()->mutable_fallback_trajectory()) =
         frame->open_space_info().chosen_paritioned_trajectory();
     auto fallback_trajectory_vec =
```
Comment (JasonZhou404) [inline modules/planning/tasks/deciders/open_space_fallback_decider.cc]: use another variable name. It's a traj and gear pair actually

Comment (deidaraho) [reply modules/planning/tasks/deciders/open_space_fallback_decider.cc]: > use another variable name. It's a traj and gear pair actually

fixed

File: modules/planning/tasks/deciders/open_space_fallback_decider.cc (hunk 1 above)
Comment (deidaraho) [reply modules/planning/tasks/deciders/open_space_fallback_decider.cc]: > Load this one from configs

I change the varaible's name, it is the distance between obstacle and vehicle

Comment (mickeyouyou) [reply modules/planning/tasks/deciders/open_space_fallback_decider.cc]: > > Load this one from configs
> 
> I change the varaible's name, it is the distance between obstacle and vehicle

It is a variable mistake about  `obstacle_to_vichcle_distance` => `obstacle_to_vehicle_distance` ? or other expressions of vehicle?

Comment (Capri2014) [reply modules/planning/tasks/deciders/open_space_fallback_decider.cc]: @mickeyouyou Thanks for the catch and I submitted a commit to fix it.

Comment (mickeyouyou) [reply modules/planning/tasks/deciders/open_space_fallback_decider.cc]: > @mickeyouyou Thanks for the catch and I submitted a commit to fix it.

👍

**Pull request merged**